package.json

```diff
@@ -4,23 +4,117 @@
   "description": "Lambda Function to Stream Logs from AWS S3 to LogDNA",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint -c .eslintrc *.js"
+    "docs": "config-doc lib/config.js > doc/env.md",
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "tap": "tap",
+    "pretest": "npm run lint",
+    "pretest:ci": "npm run lint",
+    "test": "npm run tap",
+    "test:ci": "tools/test-ci.sh",
+    "package": "tools/package.sh",
+    "release": "semantic-release"
+  },
+  "release": {
+    "branches": [
+      "master"
+    ],
+    "plugins": [
+      [
+        "@semantic-release/github",
+        {
+          "assets": [
+            {
+              "path": "pkg/logdna-s3.zip"
```

Comment:
we probably need to exclude this directory in `.gitignore` so it doesn't get added/committed as part of the release commit

Reply:
@mdeltito, I added `**/*.zip` to `.gitignore` which handles this too